src/orange/univ/UniverseVisitor.hh

```diff
@@ -0,0 +1,85 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file orange/univ/UniverseVisitor.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/math/Algorithms.hh"
+#include "orange/OrangeData.hh"
+
+#include "UniverseTypeTraits.hh"
```

Comment:
You'll need to include both `RectArrayTracker.hh` and `SimpleUnitTracker.hh` here.